samples/js/node/web/avatar/README.md

```diff
@@ -0,0 +1,96 @@
+# Instructions to run Microsoft Azure TTS Talking Avatar sample code
+
+### Basic Sample
+This sample demonstrates the basic usage of Azure text-to-speech avatar real-time API.
+
+* Step 1: Open a console and navigate to the folder containing this README.md document.
+    * Set below environment variables:
+        * `SPEECH_REGION` - the region of your Azure AI Service resource, e.g. westus2. (Azure AI Speech Toolkit: Run the Sample App will set environment variable automatically, or you can set it manually.)
+        * `SPEECH_KEY` - the key of your Azure AI Service resource.
+        * `SPEECH_PRIVATE_ENDPOINT` - (Optional) the private endpoint of your Azure speech resource. e.g. https://my-speech-service.cognitiveservices.azure.com. This is optional, and only needed when you want to use private endpoint to access Azure speech service. This is optional, which is only needed when you are using custom endpoint.
+    * Set below environment variables if you want to use customized ICE server:
+        * `ICE_SERVER_URL` - (Optional) the URL of your customized ICE server.
```

Comment:
Since this is optional, please make sure the sample code can work correctly without setting this